providers/unix/components/process/process.go

```diff
@@ -70,29 +75,6 @@ func (p *Process) start(ctx context.Context) error {
 		Setsid: true, // Run in background.
 	}
 
-	// Forward environment.
```

Comment:
Since we are not adding environment variables to send to `supervise`, I *think* that we can just rely on default environment inheritance.

Reply:
OK, I _think_ I understand this change. Is the idea that you've moved this out of the process starting logic and into the supervisor? But the behavior is otherwise pretty much the same? If so, that seems reasonable to me. I think I did it this way originally to keep the supervisor simpler and less likely to fail, but it has a basic error reporting mechanism, so it should be ok. Tho the more code in there, the more chances for badly handled panics.

Was there something that motivated this change?

Reply:
The reason for this change is that we were previously adding anything from `p.Environment` to the `supervise` process. In most cases this would be fine, but since we are running `exo`, we need to pass an `EXO_HOME` environment variable. If this variable is loaded by `supervise`, it does the wrong thing (and crashes).

Reply:
Aaaah, I gotcha now. That makes sense. To restate: You're delaying the environment application to the actual supervised process, not the supervisor as well. Sounds good 👍🏻